Sources/Client/Model/Message/MessageRead.swift

```diff
@@ -38,8 +38,4 @@ public struct MessageRead: Decodable, Hashable {
     public static func == (lhs: MessageRead, rhs: MessageRead) -> Bool {
         lhs.user == rhs.user
     }
-    
-    public func hash(into hasher: inout Hasher) {
-        hasher.combine(user)
-    }
```

Comment:
This is custom implementations, why did we remove it?

Reply:
We don't use it as `Hashable`.